docker-hub/api/dvp.yaml

```diff
@@ -0,0 +1,564 @@
+openapi: 3.0.0
+info:
+  title: DVP Data API
+  version: 1.0.0
+  x-logo:
+    url: https://docs.docker.com/images/logo-docker-main.png
+    href: /reference
+  description: |
+    The Docker DVP Data API allows Docker Verified Publishers to view image pull analytics data for their namespaces
+servers:
+  - url: https://hub-stage.docker.com/api/dvp/v1
```

Comment:
This contains an URL for staging; probably needs to be updated

Reply:
I'm not actually sure if that gets rendered anywhere, but will update in case.